drag.cc

```diff
@@ -244,8 +241,11 @@ void Dragging_info::paint(
 ) {
 	if (!rect.w)            // Not moved enough yet?
 		return;
-	if (save)           // Save background.
-		gwin->get_win()->get(save, rect.x, rect.y);
+	if (save) {          // Save background.
+        Image_buffer* tmp;
+		gwin->get_win()->get(tmp, rect.x, rect.y);
+        save.reset(tmp);
+    }
```

Comment:
Missed this one on comment above.

Reply:
I've re-read that code, I'll revert that change, as it seems to me the `save` field is either modified someplace else or is completetly redundant; I need to double check that, sorry for the hassle.